### PR TITLE
docs: change README

### PR DESCRIPTION
**Pull request by Dwipasca**
change or update README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,13 +24,49 @@ Feel free for:
...
+- go to the project directory
+
+```
+  cd chakra-icons
+```
+
+- install dependencies
+
+```
+  yarn | yarn install | npm install
```
Comment (ri7nz) [inline README.md:44]: cause we use `yarn`. don't need `npm install`

File: README.md [hunk 2]
```diff
@@ -24,13 +24,49 @@ Feel free for:
 
 - Create Issue, Request Features, & send a pull request.
 
-### Make new packages
+## How to use
```
Comment (ri7nz) [inline README.md:27]:
```suggestion
## How to add new icons
```

File: README.md (hunk 1 above)
Comment (ri7nz) [reply README.md:44]:
```suggestion
  yarn | yarn install
```

File: README.md [hunk 3]
```diff
@@ -24,13 +24,49 @@ Feel free for:
...
-- them run build `yarn build`
-- when success and no issues, you can make pull Request
+
+- build the new package
+
+```
+  yarn build
+```
+
+- if the process build success and no issue, then you can make a new PR
```
Comment (ri7nz) [inline README.md:69]:
```suggestion
- if the process build success and no issue, then you can make a new [PRs](https://github.com/kodingdotninja/chakra-icons/compare)
```

File: README.md (hunk 1 above)
Comment (Dwipasca) [reply README.md:44]: oke I got it

**Pull request closed without merging**